pydrive2/fs/spec.py

```diff
@@ -245,17 +245,18 @@ def split_path(self, path):
     @wrap_prop(threading.RLock())
     @cached_property
     def _ids_cache(self):
-        cache = {
-            "dirs": defaultdict(list),
-            "ids": {},
-            "root_id": self._get_item_id(
-                self.path,
-                use_cache=False,
-                hint="Confirm the directory exists and you can access it.",
-            ),
-        }
+        cache = {"dirs": defaultdict(list), "ids": {}}
```

Comment:
C: had to cleanup this a bit (root id is not used anymore, also we now cache `ids` not a single `id` even for the base